### PR TITLE
Update email validation template

### PR DESCRIPTION
**Pull request by tiagograf-AOT**
## Status
**Ready**

## Fixes or Implements [AB#8028](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8028)
- [x] Enhancement
- [ ] Bug
- [ ] Documentation

## Description:
Updates the Email verification template.

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

### CODE REVIEW

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_369)

File: Apps/Database/src/Migrations/20200304214641_UpdateVerifyEmailTemplate.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
+﻿using System;
```
Comment (FuriousLlama) [inline Apps/Database/src/Migrations/20200304214641_UpdateVerifyEmailTemplate.cs]: copyright missing

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_369)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_369)

**Pull request merged**